### DISCUSSION TITLE
How to get decoration and tooltip in ts/js file by VSCode extension ?

### DISCUSSION THREAD

**Discussion by wzc520pyfm** [unocss/unocss#3600] [Q&A] [answered]
I writing shortcuts in `.ts`, but i cann't get decoration and tooltip😫.

<img width="630" alt="image" src="https://github.com/unocss/unocss/assets/69044080/8a43717f-77ba-4418-9f09-67c26a8a0cc8">

Expect：Has decoration and tooltip, like `.vue`

<img width="458" alt="image" src="https://github.com/unocss/unocss/assets/69044080/b1844559-5cef-4b6e-b33f-29405964e5e1">

**Comment by zyyv** [comment, marked as answer]
You can also add `@unocss-include` magic comment, per-file basis, anywhere in the file that you want UnoCSS to scan. If you need to scan `.js` or `.ts` files, add them in the configuration to include all `js/ts` files as scan targets.

**Reply by wzc520pyfm** [reply]
Thanks! This worked for me. 
```ts
export default defineConfig({
  content: {
    pipeline: {
      include: [/\.(vue|svelte|[jt]sx|mdx?|astro|elm|php|phtml|html|ts|js)($|\?)/],
    },
  },
})
```
And, i think it's worth noting：`uno.config.ts` needs to be in the root dir, i made this mistake when using monorepo.
